experiments/tutorials/train_tiny_model_cpu.py

```diff
@@ -40,18 +37,12 @@
 
 # 2. Tokenize the dataset with sampling
 # For this tutorial, we limit to 1000 documents per shard
-tinystories_tokenized = ExecutorStep(
-    name=os.path.join("tokenized", tinystories_hf_id),
-    description=f"Tokenize TinyStories with 1000 samples per shard using {marin_tokenizer}",
-    fn=tokenize,
-    config=HfTokenizeConfig(
-        id=tinystories_hf_id,
-        cache_path=this_output_path(),
-        tokenizer=ensure_versioned(marin_tokenizer),
-        format=TextLmDatasetFormat(),
-        sample_count=versioned(1000),
-    ),
-    pip_dependency_groups=["tokenize_train"],
+tinystories_tokenized = default_tokenize(
```

Comment:
Not new to this PR, but... I don't like the default_* pattern. I find it obscures the underlying mechanism, and sort of "squirrels away" potentially bad interfaces instead of trying to refactor or improve them.

In this case what's the advantage of using default_train here versus exposing the ExecutorStep? Just because it's the (no pun intended) default? :)

Reply:
the syntax is kinda scary imho

Reply:
The original intention of `default_*` if to have a sane default way of doing things, so that if you're working on the dataset, you can just call `default_train` and not have to worry about configuring the right training config.  I think since then, it sort of has lost its meaning and is just an all-purpose wrapper.